e2e/game_page.js

```diff
@@ -1,6 +1,15 @@
 "use strict";
 import { Selector } from 'testcafe'; // first import testcafe selectors
 
+async function countCardsInRow (rows) {
```

Comment:
Apparently this function is returning `{}`.
It seems to enter only once to the `for`.